poetry.lock

```diff
@@ -1,15 +1,14 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
-
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
This file is autogenerated, but it is replacing a newer gen tool with an older one, 
latest Poetry is `1.8.3` https://github.com/python-poetry/poetry/releases/tag/1.8.3
maybe is best to always move forwards in semver not backwards IMHO.